.github/workflows/builds.yml

```diff
@@ -786,6 +786,7 @@ jobs:
                 ccache \
                 clang \
                 diffutils \
+                dpdk-devel \
```

Comment:
commit does 2 related but different things:
1. expand CI dpdk coverage
2. add tests
Feel this should be split.